### ISSUE TITLE
[feature request] the El input component V-model does not support the lazy modifier

### ISSUE THREAD

**Issue by luoyi58624** [element-plus/element-plus#3997]
### Existing Component
Yes

### Component Name
1.1.0-beta.24

### Description
在默认情况下，v-model 在每次 input 事件触发后将输入框的值与数据进行同步，你可以添加 lazy 修饰符，从而转为在 change 事件之后进行同步，使用原生input添加lazy修饰符有效，但el-input无效！
原文链接：https://v3.cn.vuejs.org/guide/forms.html#lazy

**Comment by winternet-studio** [comment]
Yes, why hasn't this been implemented yet? Issue was opened 4 years ago...!
